app/controllers/users_controller.rb

```diff
@@ -0,0 +1,70 @@
+#rubocop:disable all
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[ show edit update destroy ]
+
+  # GET /users or /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1 or /users/1.json
+  def show
+  end
+
+  # GET /users/new
+  def new
+    @user = User.new
+  end
+
+  # GET /users/1/edit
+  def edit
+  end
+
+  # POST /users or /users.json
+  def create
+    @user = User.new(user_params)
+
+    respond_to do |format|
+      if @user.save
+        format.html { redirect_to @user, notice: "User was successfully created." }
+        format.json { render :show, status: :created, location: @user }
```

Comment:
- [ ] As described on the project requirements. Please render the new user page after a user is saved correctly. 🙏 😄
![user_create](https://user-images.githubusercontent.com/47625407/128914568-deac4c5a-e596-4b25-a4ef-814349d8d92f.png)